src/js/components/put-away/PutAwaySecondPage.jsx

```diff
@@ -504,10 +504,10 @@ class PutAwaySecondPage extends Component {
    */
   generatePutAwayList() {
     this.props.showSpinner();
-    const url = '/putAway/generatePdf';
+    const url = `/putAway/generatePdf/${this.state.putAway.id}`;
```

Comment:
As discussed at the beginning of the migration - whenever we touch the urls, we should add them to our urls factory.
Would you mind adding it there?

Reply:
sure, will do